src/library/Uno.Themes.WinUI.Markup/Theme/Typography.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+using Windows.UI.Text;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class Typography
+		{
+			public static class Resources
+			{
+				public static class DisplayLarge
```

Comment:
Maybe split into Display -> Large? @kazo0

Reply:
Let's keep it as matching the actual style name that we use on the TextBlocks: `DisplayLarge`, `DisplayMedium`, etc.